dosomething/fastly-frontend/legacy_redirects_recv.vcl

```diff
@@ -0,0 +1,14 @@
+# Redirect */actnow/* page requests.
+if (req.url ~ "(?i)^\/((us|mx|br)\/)?actnow\/") { error 800 "/us"; }
+
+# Redirect */programs/* page requests.
+if (req.url ~ "(?i)^\/((us|mx|br)\/)?programs\/") { error 800 "/us"; }
+  
+# Redirect */project/* page requests.
+if (req.url ~ "(?i)^\/((us|mx|br)\/)?project(s)?\/") { error 800 "/us"; }
+
+# Redirect */about/press/* page requests.
+if (req.url ~ "(?i)^\/((us|mx|br)\/)?about\/press\/") { error 800 "/us/about/our-press"; }
+
+# Redirect */about/team/* page requests.
+if (req.url ~ "(?i)^\/((us|mx|br)\/)?about\/team\/") { error 800 "/us/about/our-people"; }
```

Comment:
One potentially small gotcha: since this regex ends in the `/`, this won't match on [`/about/team`](https://www.dosomething.org/us/about/team).

Are we getting any hits there?

Reply:
Not that I saw, but also don't want to target for not ending in `/` since we may have some page in the future that begins with team and is hyphenated. We can just do an Aurora redirect for this one.

Reply:
Oh sure, Aurora redirect works for me! 👍

Reply:
Added the redirect to Aurora to help :)